pages/__tests__/rotas.test.js

```diff
@@ -0,0 +1,31 @@
+import { mount, createLocalVue } from "@vue/test-utils"
+import Vuetify from 'vuetify'
+
+import RoutesPage from "~/pages/rotas.vue"
+
+describe ("Pages / RoutesPage", ()=>{
+  const defaultData={
+    cidsRoutes: null,
+    model: 0
+  }
+
+  const localVue = createLocalVue()
+  let vuetify
+
+  beforeEach(() => {
+    vuetify = new Vuetify()
```

Comment:
precisa mesmo instanciar o vuetify? qualquer coisa tenta testar com `shallowMount`

Reply:
Aqui precisa, orientações da doc. 
Tentei sem, mas ai o jest aparentemente n enxerga o vuetify

`https://vuetifyjs.com/en/getting-started/unit-testing/#spec-tests:~:text=()%0A%20%20let%20vuetify-,beforeEach(()%20%3D%3E%20%7B%0A%20%20%20%20vuetify%20%3D%20new%20Vuetify()%0A%20%20%7D),-it(%27should%20work`